HousingSearchApi/V1/Interfaces/SearchGateway.cs

```diff
@@ -62,5 +66,27 @@ public async Task<GetAssetListResponse> GetListOfAssets(HousingSearchRequest que
 
             return assetListResponse;
         }
+
+        public async Task<GetTransactionListResponse> GetListOfTransactions(GetTransactionSearchRequest request)
+        {
+            var searchRequest = new HousingSearchRequest
+            {
+                SearchText = request.SearchText,
+                Page = request.Page,
+                PageSize = request.PageSize
+            };
+
+            var searchResponse = await _elasticSearchWrapper.Search<QueryableTransaction>(searchRequest).ConfigureAwait(false);
+
+            var loadedCount = searchResponse.Documents.Count;
```

Comment:
can you combine `loadedCount` and `searchResponse.Documents.Select(queryableTransaction =>
                queryableTransaction.Create()` and pass them into the `GetTransactionListResponse` using `Create()` - or pass in `searchResponse` when creating the object and move all this logic into that class?